scripts/validation/validators.py

```diff
@@ -136,6 +140,68 @@ def check_duplicate_contract(glob_path: str):
         raise ValidationError(f"Invalid files: {errors}")
 
 
+def check_duplicate_plugin(glob_path: str):
+    invalid, errors, results, all = 0, [], [], {}
+    for filename in glob.iglob(glob_path, recursive=True):
+        logger.debug("Validating %s... for duplicate plugins", filename)
+        with open(filename, "r", newline="") as f:
+            data = f.read()
+
+        try:
+            target_data = json.loads(data)
+        except JSONDecodeError as err:
+            logger.debug(
+                "\tinvalid: File %s is not a valid json", filename, exc_info=True
+            )
+            errors.append({"file": filename, "message": str(err)})
+            continue
+
+        records = []
+        blockchain = target_data.get("blockchainName", None)
+        for contract in target_data.get("contracts", []):
+            address = contract.get("address", None)
+            if address is None:
+                continue
+            elif address in __excluded_contracts_from_plugin_collision:
+                continue
+
+            address = address.lower()
+
+            for selector, data in contract.get("selectors", []).items():
+                plugin = data["plugin"]
+
+                record = {
+                    "filename": filename,
+                    "blockchain": blockchain,
+                    "contract": address,
+                    "selector": selector,
+                    "plugin": plugin,
+                }
+                logger.debug(f"adding record {record}")
+                records.append(record)
+
+        # check if among the records there are multiple plugins associated with a given (chain_id, contract, selector) triplet
+        for record in records:
+            filename = record["filename"]
+            blockchain = record["blockchain"]
+            contract = record["contract"]
+            selector = record["selector"]
```

Comment:
I suggest to lowercase contract + selector here

Reply:
Ok I added a lowercase when saving them in the `records` big list